modules/dragster/src/main/ts/ephox/dragster/file/FileList.ts

```diff
@@ -1,7 +1,11 @@
 import { Arr } from '@ephox/katamari';
 
 const createFileList = (inputFiles: File[]): FileList => {
-  const files = {
+  const files: {
```

Comment:
Can this be just FileList since the object is created it should not clash with the readonly property of the length. Or is the null stuff from that type annoying here?